src/com/theironyard/novauc/Array1.java

```diff
@@ -0,0 +1,32 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by octavio on 2/23/17.
+ */
+
+//        We'll say that a 1 immediately followed by a 3 in an array is an "unlucky" 1. Return true if the given array contains
+//        an unlucky 1 in the first 2 or last 2 positions in the array.
+//
+//        unlucky1([1, 3, 4, 5]) → true
+//        unlucky1([2, 1, 3, 4, 5]) → true
+//        unlucky1([1, 1, 1]) → false
+
+
+
+public class Array1 {
+
```

Comment:
You can write all of the problems you are trying to solve in one class.  Please write tests for these though and resubmit.